website/src/categories/components/combobox/code.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Combobox code
+tags: code
+APIdescription: {
+  sl-combobox: "DESCRIPTION"
+}
+eleventyNavigation:
+  parent: Combobox
+  key: ComboboxCode
+---
+
+<section>
+
+<div class="ds-example">
+
+<sl-combobox multiple value='["0","3"]'>
```

Comment:
In the example last tag is being clipped

<img width="958" alt="image" src="https://github.com/user-attachments/assets/9d48cd94-bbf9-4ee0-8459-36467dab750a" />